app/kumactl/cmd/inspect/inspect_zones.go

```diff
@@ -75,6 +75,13 @@ func printZoneOverviews(now time.Time, zoneOverviews *system.ZoneOverviewResourc
 				}
 				lastUpdated := util_proto.MustTimestampFromProto(lastSubscription.GetStatus().GetLastUpdateTime())
 
+				remoteCpVersion := "-"
+				if lastSubscription.GetVersion() != nil {
+					if lastSubscription.Version.KumaCp != nil {
```

Comment:
You can simplify these `if` statements with nil-safe getters:
```
var remoteCpVersion string
if remoteCpVersion = lastSubscription.GetVersion().GetKumaCp().GetVersion(); remoteCpVersion == "" {
        remoteCpVersion = "-"
}
```

Reply:
I changed it to an empty string to match it with `inspect dataplanes`.